ezra-project.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Ezra Project
+Comment=ezra-project
+Exec=ezra-project
+Icon=ezra-project
```

Comment:
Good reminder that we still need a nice icon for Ezra Project! :)

Reply:
See #26.